Source/WebKit/WebProcess/cocoa/VideoPresentationManager.mm

```diff
@@ -732,6 +732,24 @@ static FloatRect inlineVideoFrame(HTMLVideoElement& element)
     ensureModel(contextId).requestRouteSharingPolicyAndContextUID(WTFMove(reply));
 }
 
+void VideoPresentationManager::ensureUpdatedVideoDimensions(PlaybackSessionContextIdentifier contextId, WebCore::FloatSize existingVideoDimensions)
+{
+    RefPtr page = m_page.get();
+    if (!page)
+        return;
+
+    auto model = std::get<0>(m_contextMap.get(contextId));
+    if (!model)
+        return;
```

Comment:
It's a shame there's not a `ModelInterfaceTuple getModelAndInterface(PlaybackSessionContextIdentifier) const`, `Model* getModel(PlaybackSessionContextIdentifier) const` and `Interface* getInterface(PlaybackSessionContextIdentifier) const`. Something for a future patch.

Reply:
Right, that would be nice. I can follow up on that. To be clear, I didn't use one of the `ensure` functions because I didn't want to create a model if one didn't exist.